addon/components/_private/attribute-editor/index.ts

```diff
@@ -7,13 +7,48 @@ import TransformUtils from '@lblod/ember-rdfa-editor/utils/_private/transform-ut
 import type { ResolvedPNode } from '@lblod/ember-rdfa-editor/utils/_private/types';
 import { Changeset, EmberChangeset } from 'ember-changeset';
 import { trackedReset } from 'tracked-toolbox';
+import { dependencySatisfies, macroCondition } from '@embroider/macros';
+import { importSync } from '@embroider/macros';
+const CheckIcon = macroCondition(
```

Comment:
Nice approach to supporting the older versions too :) I wonder if it is useful to move these imports to a seperate general file to avoid code duplication.

Reply:
I thought about that, but it would mean we maintain a separate file or files with all of the icon imports we use, as they are all in separate files. Perhaps it would be cleaner though. I'm not sure.

Reply:
Yeah it's not super necessary indeed. The icon compatibility layer will not be for the long term anyway.